static/app/views/explore/spans/spansTab.tsx

```diff
@@ -81,64 +81,227 @@ export function SpansTabOnboarding({
 }: SpansTabOnboardingProps) {
   return (
     <Layout.Body>
-      <TopSection>
-        <FilterSection>
-          <StyledPageFilterBar condensed>
-            <ProjectPageFilter />
-            <EnvironmentPageFilter />
-            <DatePageFilter
-              defaultPeriod={defaultPeriod}
-              maxPickableDays={maxPickableDays}
-              relativeOptions={({arbitraryOptions}) => ({
-                ...arbitraryOptions,
-                ...relativeOptions,
-              })}
-            />
-          </StyledPageFilterBar>
-        </FilterSection>
-      </TopSection>
+      <PageFilterBar condensed>
+        <ProjectPageFilter />
+        <EnvironmentPageFilter />
+        <DatePageFilter
+          defaultPeriod={defaultPeriod}
+          maxPickableDays={maxPickableDays}
+          relativeOptions={({arbitraryOptions}) => ({
+            ...arbitraryOptions,
+            ...relativeOptions,
+          })}
+        />
+      </PageFilterBar>
       <OnboardingContentSection>
         <Onboarding project={project} organization={organization} />
       </OnboardingContentSection>
     </Layout.Body>
   );
 }
 
-type SpanTabProps = PickableDays;
+interface SpanTabProps extends PickableDays {}
 
 export function SpansTabContent({
   defaultPeriod,
   maxPickableDays,
   relativeOptions,
 }: SpanTabProps) {
   const organization = useOrganization();
-  const {selection} = usePageFilters();
-  const mode = useExploreMode();
-  const visualizes = useExploreVisualizes();
-  const setVisualizes = useSetExploreVisualizes();
-  const fields = useExploreFields();
-  const [samplesTab, setSamplesTab] = useTab();
-
-  const {tags: numberTags, isLoading: numberTagsLoading} = useSpanTags('number');
-  const {tags: stringTags, isLoading: stringTagsLoading} = useSpanTags('string');
+  const [controlSectionExpanded, setControlSectionExpanded] = useState(true);
 
-  const query = useExploreQuery();
-  const setExplorePageParams = useSetExplorePageParams();
+  return (
+    <Fragment>
+      <VisitExploreTrigger />
+      <BodySearch>
+        <SpanTabSearchSection
+          defaultPeriod={defaultPeriod}
+          maxPickableDays={maxPickableDays}
+          relativeOptions={relativeOptions}
+        />
+      </BodySearch>
+      <BodyContent>
+        <SpanTabControlSection
+          organization={organization}
+          controlSectionExpanded={controlSectionExpanded}
+        />
+        <SpanTabContentSection
+          maxPickableDays={maxPickableDays}
+          organization={organization}
+          setControlSectionExpanded={setControlSectionExpanded}
+          controlSectionExpanded={controlSectionExpanded}
+        />
+      </BodyContent>
+    </Fragment>
+  );
+}
 
+function VisitExploreTrigger() {
   const id = useExploreId();
   const visitQuery = useVisitQuery();
   useEffect(() => {
     if (defined(id)) {
       visitQuery(id);
     }
   }, [id, visitQuery]);
+  return null;
+}
```

Comment:
Why does this need to be a component? Can't we also just make it a hook, since it returns nothing but composes other hooks